RetailCoder.VBE/UI/RubberduckUI.de.resx

```diff
@@ -908,13 +908,13 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren. Die Originaldatei wird
     <value>Standard Repositorypfad</value>
   </data>
   <data name="SourceControl_EmailAddressLabel" xml:space="preserve">
-    <value>Email Adresse</value>
+    <value>E-Mail-Adresse</value>
   </data>
   <data name="SourceControl_GlobalSettings" xml:space="preserve">
     <value>Globale Einstellungen</value>
   </data>
   <data name="SourceControl_IgnoreFile" xml:space="preserve">
-    <value>Datei ignorieren</value>
+    <value>'.gitignore' Datei</value>
```

Comment:
@Vogel612 As I see these both buttons on the global settings next to each other I think only .gitignore and gitattributes are ment. Or what is "Arribute File" if "Ignore File" is "ignore this file"?